toolkits/slack/arcade_slack/models.py

```diff
@@ -174,7 +172,7 @@ class Message(SlackMessage, total=False):
     datetime_timestamp: str  # Human-readable datetime string (e.g. "2025-01-22 12:00:00")
 
 
-class ConversationMetadata(TypedDict, total=True):
+class ConversationMetadata(TypedDict, total=False):
```

Comment:
@byrro I've removed the `typing_extensions` dependency. Please let me know if these changes maintain your intended behavior.